src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/ProjectRazorJson.cs

```diff
@@ -2,41 +2,14 @@
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using Microsoft.AspNetCore.Razor;
 using Microsoft.AspNetCore.Razor.Language;
-using Microsoft.CodeAnalysis.Razor.ProjectSystem;
 using Newtonsoft.Json;
 
-namespace Microsoft.CodeAnalysis.Razor.Workspaces.Serialization;
+namespace Microsoft.AspNetCore.Razor.ProjectEngineHost.Serialization;
 
 [JsonConverter(typeof(ProjectRazorJsonJsonConverter))]
 internal sealed class ProjectRazorJson
 {
-    public ProjectRazorJson(string serializedFilePath, IProjectSnapshot project)
```

Comment:
The link between `IProjectSnapshot` and `ProjectRazorJson` was just used to get a couple of properies, so I converted this constructor to an extension method, so `ProjectRazorJson` can be dependency free.